src/app/portlets/dot-service-integration/dot-service-integration-configuration/dot-service-integration-configuration-list/dot-service-integration-configuration-item/dot-service-integration-configuration-item.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { DotServiceIntegrationSites } from '@shared/models/dot-service-integration/dot-service-integration.model';
+import { DotAlertConfirmService } from '@services/dot-alert-confirm';
+
+import { DotMessageService } from '@services/dot-messages-service';
+import { take } from 'rxjs/operators';
+
+@Component({
+    selector: 'dot-service-integration-configuration-item',
+    templateUrl: './dot-service-integration-configuration-item.component.html',
+    styleUrls: ['./dot-service-integration-configuration-item.component.scss']
+})
+export class DotServiceIntegrationConfigurationItemComponent implements OnInit {
+    @Input() site: DotServiceIntegrationSites;
+
+    @Output() goto = new EventEmitter<DotServiceIntegrationSites>();
```

Comment:
Even tho what's happen with this event and the end is going to a configuration detail, this components shouln't matter that, this should be like `edit`